machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -6,37 +6,71 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-For AWS, {product-title} clusters that are created with version 4.12 or later use the Control Plane Machine Set Operator by default. 
+The process for getting started with the Control Plane Machine Set Operator depends on the state of the `ControlPlaneMachineSet` custom resource (CR) in your cluster. 
 
-To use this Operator on AWS clusters that are upgraded to {product-title} version 4.12 or later from an earlier version, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an active generated CR:: Clusters that have a generated CR with an active state use the Operator by default. No administrator action is required. 
+//+
+//AWS clusters that are created with {product-title} version 4.12 or later use the Control Plane Machine Set Operator by default. 
 
-For Azure and VMware vSphere clusters, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[create a `ControlPlaneMachineSet` custom resource (CR)] and then xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an inactive generated CR:: For clusters that include an inactive generated CR, you must review the CR configuration and xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
```

Comment:
```suggestion
Clusters with an inactive generated CR:: For clusters that include an inactive generated CR, you must review the CR configuration and xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Resource].
```
I'd probably go with activate the Resource here. The Operator is always running and computing diffs between current and desired state (based on Control Plane Machines vs (all states) CPMS spec) but won't act on it until the Resource is Active.